ee/ucp/admin/install/install-on-azure.md

```diff
@@ -36,31 +36,29 @@ There are two options for provisoning IPs for the Kubernetes cluster on Azure
 
 ## Azure Prerequisites 
 
-You must meet these infrastructure prerequisites in order 
-to successfully deploy Docker UCP on Azure
-
-- All UCP Nodes (Managers and Workers) need to be deployed into the same 
-Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
-components could be deployed in a second Azure Resource Group.
-- The Azure Vnet and Subnet must be appropriately sized for your
-  environment, and addresses from this pool are consumed by Kubernetes Pods. For more information, see
-  [Considerations for IPAM
+You must meet the following infrastructure prerequisites in order 
+to successfully deploy Docker UCP on Azure:
+
+- All UCP worker and manager nodes need to be deployed into the same Azure
+  Resource Group. The Azure Networking (Vnets, Subnets, Security Groups)
+  components could be deployed in a second Azure Resource Group.
+- The Azure Vnet and Subnet must be appropriately sized for your environment, as
+  addresses from this pool will be consumed by Kubernetes Pods. For more
+  information, see [Considerations for IPAM
   Configuration](#considerations-for-ipam-configuration).
-- All UCP Nodes (Managers and Workers) need to be attached to the same 
-Azure Subnet.
-- All UCP (Managers and Workers) need to be tagged in Azure with the 
-`Orchestrator` tag. Note the value for this tag is the Kubernetes version number
-in the format `Orchestrator=Kubernetes:x.y.z`. This value may change in each 
-UCP release. To find the relevant version please see the UCP 
-[Release Notes](../../release-notes). For example for UCP 3.1.0 the tag 
-would be `Orchestrator=Kubernetes:1.11.2`. 
-- The Azure Virtual Machine Object Name needs to match the Azure Virtual Machine 
-Computer Name and the Node Operating System's Hostname. Note this applies to the 
-FQDN of the host including domain names. 
-- An Azure Service Principal with `Contributor` access to the Azure Resource 
-Group hosting the UCP Nodes. Note, if using a separate networking Resource 
-Group the same Service Principal will need `Network Contributor` access to this 
-Resource Group.
+- All UCP worker and manager nodes need to be attached to the same Azure
+  Subnet.
+- All UCP workers and managers need to have the `Orchestrator` tag in Azure with the Kubernetes version as the value, following this format: `Orchestrator=Kubernetes:x.y.z`. This value may change in
+  each UCP release. To find the relevant version please see the UCP [Release
+  Notes](../../release-notes). For example for UCP `3.1.0` the tag would be
+  `Orchestrator=Kubernetes:1.11.2`. 
+- The Azure Virtual Machine Object Name needs to match the Azure Virtual Machine
```

Comment:
I would like to add one more requirement for Azure VM name and OS hostname: _the name must be all lowercased_. In k8s world, the node name will be converted to lowercases, causing mismatch with Azure VM name which is not lowercased. We had customer cases for this issue.

Just curious, what's the difference between `Azure Virtual Machine Object Name` and `Azure Virtual Machine Computer Name`?

Reply:
I've added the lowercase comment to the pre-reqs. 

I may have named these wrong but Object Name (found in resource ID) and the Virtual Machine Computer name can be different. I've often seen these different as part of ARM template deployments. @stevenfollis does this sound right?

The Kubernetes code is first checking the ID, I.e 

```
"id": "/subscriptions/<sub-i>/resourceGroups/<resource-group>/providers/Microsoft.Compute/virtualMachines/manager-01"
```

I think it gets this value from the VM hostname. The Azure Computer name by default is the OS Hostname.

In the Portal - Red Line = ID, Blue Line = Computer Name

![image](https://user-images.githubusercontent.com/10342453/55231547-dd7b8d80-521a-11e9-9b12-31aec1fba609.png)